src/design-system/PageColumn.tsx

```diff
@@ -3,6 +3,10 @@ import styled from "styled-components";
 export const PageContainer = styled.div`
   display: flex;
   flex-direction: row;
+
+  @media (max-width: 700px) {
+    flex-direction: column;
+  }
 `;
 export const Column = styled.div<{ width?: string }>`
   margin: 20px;
```

Comment:
I think we should also change the default width of the `Column` on line 13 to be `100%` within that same media query

Reply:
@daschi good catch, thanks! How about 95% since we lose some of the right margin? (I eyeballed 95%, can see 100% effect in this gif below.)

![responsive3](https://user-images.githubusercontent.com/1372946/81618990-7036a780-939d-11ea-8cf0-08e531ee477b.gif)

Reply:
oh yeah--actually it might work better if you change it to `width: unset` and then it should fit the space I think

Reply:
OK - `unset` doesn't work in IE11, but `inherit` does which I think would do the same thing here

Reply:
👍 nice, never thought of using inherit for width, that's cool! made the fix and looks good on facility page as well